packages/lighthouse-service/src/audit-manager/typedef.graphql

```diff
@@ -26,6 +26,7 @@ type LighthouseProjectType {
   slug: String
   externalUrl: String
   token: ID
+  adminToken: ID
```

Comment:
Should the admin token to be exposed in graphql.

Reply:
Its used by Dev Console config screen and that too when a project is created.